src/ingest_validation_tools/table-schemas/assays/scatacseq-v1.yaml

```diff
@@ -58,13 +58,7 @@ fields:
       - bulkATACseq
       - snATACseq
       - sciATACseq
-- name: transposition_transposase_source
-  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
```

Comment:
description: The source of the Tn5 transposase used for capturing accessible chromatin.